src/model/MintInfo.ts

```diff
@@ -92,6 +95,17 @@ export class MintInfo {
 		return { supported: false };
 	}
 
+	private checkNut19() {
+		if (this._mintInfo.nuts[19] && this._mintInfo.nuts[19]?.cached_endpoints.length > 0) {
+			return {
+				supported: true,
+				ttl: this._mintInfo.nuts[19].ttl,
+				params: this._mintInfo.nuts[19].cached_endpoints
+			};
+		}
+		return { supported: false };
+	}
+
```

Comment:
This looks solid - requiring at least one cached endpoint for the supported flag makes sense. If we adopt a stricter NUT19Policy type definition as per my suggestion, we can map `null` to `Infinity` for indefinite caching, eg:

```ts
private checkNut19() {
  const rawPolicy = this._mintInfo.nuts[19];
  if (rawPolicy && rawPolicy.cached_endpoints.length > 0) {
    return {
      supported: true,
      ttl: rawPolicy.ttl ?? Infinity, // Normalize null to Infinity for indefinite caching
      params: rawPolicy.cached_endpoints
    };
  }
  return { supported: false };
}
```